### PR TITLE
Add instructions on how to report security issues

### PR DESCRIPTION
**Pull request by seb128**
Quoting SEC0026 - SSDLC

> SECURITY.md (V1.1)
> Any public repository must also include a SECURITY.md file in the root directory, which educates users/contributors on
> how to report a security concern.

The content of the file is based on the template referenced in the spec and what other Canonical repository are already using

### CODE REVIEW

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# How to report a security issue with authd-msentraid
```
Comment (didrocks) [inline SECURITY.md:1]: This is generic and not only the msentra id brokers. Do you mind rephrasing that this is for any brokers here?

Or this line could be with "authd oidc-based broker" and then, we can have one line listing them (authd-msentraid for now).

**Review by didrocks — CHANGES_REQUESTED** [review]
Once small comments/adjustement that I think is necessary for this monorepo with different project targets before we merge it. Tell me what you think?

File: SECURITY.md (hunk 1 above)
Comment (seb128) [reply SECURITY.md:1]: It is targetting the [msentraid](https://github.com/ubuntu/authd-oidc-brokers/tree/msentraid) branch and matching the name set in the snapcraft.yaml but I'm fine changing it to be generic if you prefer

Comment (didrocks) [reply SECURITY.md:1]: Right, I don’t think people will really go to each branch to read about security.md for that specific branch, also GitHub integrates when you open an issue a read of `security.md` which seems to be following the content in HEAD (like it does for the bug template). So I would:
1. Target only the main branch (automerge will copy them in dedicated branches anyway)
2. Have generic content there, that will cover all brokers

**Review by didrocks — APPROVED** [review]
Looks good now! Thanks!

**Pull request merged**